prober/http.go

```diff
@@ -411,6 +424,8 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 		ConnectDone:          tt.ConnectDone,
 		GotConn:              tt.GotConn,
 		GotFirstResponseByte: tt.GotFirstResponseByte,
+		TLSHandshakeStart:    tt.TLSHandshakeStart,
```

Comment:
this got introduced in Go 1.8, which is fine, as we are already requiring at least Go 1.13